cfn-resources/cluster/mongodb-atlas-cluster.json

```diff
@@ -446,7 +450,8 @@
   "createOnlyProperties": [
     "/properties/Name",
     "/properties/ProjectId",
-    "/properties/Profile"
+    "/properties/Profile",
+    "/properties/GlobalClusterSelfManagedSharding"
```

Comment:
q: Does defining this property as createOnly have any impact when `GlobalClusterSelfManagedSharding` is left empty and then populated with value coming from API?

Reply:
The CFN implementation differs a bit from Terraform:
1. If the value is `nil` then we use the value from the API
2. else, we use the value specified in the model

The normal test case/examples don't specify this flag and still managed the full CRUD operations, so I believe we should be safe 👍